src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/Compostable.java

```diff
@@ -11,13 +11,15 @@
 /**
  * Data map value for {@linkplain NeoForgeDataMaps#COMPOSTABLES compostables}.
  *
- * @param chance the chance that a compost is successful
+ * @param chance             the chance that a compost is successful
+ * @param canVillagerCompost whether farmer villagers can compost the item
  */
-public record Compostable(float chance) {
+public record Compostable(float chance, boolean canVillagerCompost) {
```

Comment:
Do we want an overload constructor that defaults to `false` for `canVillagerCompost` when just passed a `chance`? I believe that would also make this PR not be a breaking change.